### ISSUE TITLE
implement CoincidenceQuery.get_reconstructions_from_station()

### ISSUE THREAD

**Issue by tomkooij** [HiSPARC/sapphire#96]
Currently retrieving (event) reconstructions from a coincidence is difficult:

```python
cq = CoincidenceQuery(data)
coincidences = cq.all([507, 510], start=start, stop=stop, iterator=True)
reconstructions_iterator = (cq._get_reconstructions(coincidence)
                            for coincidence in coincidences)
coincidences_events = (cq._events_from_stations(reconstructions, [507, 510])
                       for reconstructions in reconstructions_iterator)
specific_reconstructions = cq.minimum_events_for_coincidence(coincidences_events, n=2)
```

A method that resembles: ```events_from_stations(coincidences,[507, 510])``` for reconstructions is needed.

**Comment by 153957** [comment]
i.e. a copy of https://github.com/HiSPARC/sapphire/blob/master/sapphire/analysis/coincidence_queries.py#L282-L294
but then for reconstructions.

**Comment by 153957** [comment]
Fixed in eb5e40fd692aeb3fc8a6d53e9d09e579d549ca74

**Comment by tomkooij** [comment]
+1
